### PR TITLE
Align icons in skillmap firefox

### PR DESCRIPTION
**Pull request by livcheerful**
fixes https://github.com/microsoft/pxt-arcade/issues/3859

The units in firefox are just different. Both are using pixels, but firefox needs more? I tried some other things but nothing worked for me other than special casing it. If there's a better way lemme know 😓

### CODE REVIEW

File: skillmap/src/lib/browserUtils.ts [hunk 1]
```diff
@@ -282,4 +282,39 @@ export function tickEvent(id: string, data?: { [key: string]: string | number })
     data["pageTitle"] = pageTitle;
 
     (window as any).pxtTickEvent?.(id, data);
-}
\ No newline at end of file
+}
+
+export function hasNavigator(): boolean {
```
Comment (eanders-MS) [inline skillmap/src/lib/browserUtils.ts:287]: why copy these methods? pxtlib's browserutils is accessible from skillmap.

Comment (livcheerful) [reply skillmap/src/lib/browserUtils.ts:287]: lol I forgot. i'll fix it

**Review by eanders-MS — APPROVED** [review]

**Comment by riknoll** [comment]
can i get a test build? this also often breaks on mac vs pc

**Comment by livcheerful** [comment]
yeah i'll spin one up! i tried to do one just now and saw something weird, so it might be a bit...but i'm working on it!

**Comment by livcheerful** [comment]
here it is!! https://arcade.makecode.com/app/87b96eebfb815fa52e9bd07975590a9647845449-d68d14a157---skillmap 

lemme know if you see bad things on mac :O

**Comment by riknoll** [comment]
Yeah, they're pretty off on mac:

Screen Shot 2021-09-08 at 9 15 48 AM

**Comment by livcheerful** [comment]
ahhhhhh how do they look normally?

**Comment by riknoll** [comment]
exactly the same! so i'm guessing the check is just failing on mac. In any case, not a regression so feel free to merge

**Comment by shakao** [comment]
does adding `dominant-baseline="middle"` to the text nodes do anything? (based on https://stackoverflow.com/questions/44545006/svg-shows-different-on-firefox-and-edge) 

also definitely a lingering bug from when this was first written, but the dy/yOffset should be calculated from the width/selectedUnit probably

**Comment by livcheerful** [comment]
ooo yeah it does i'll push it. Richard i'll also make another upload for mac??

**Comment by livcheerful** [comment]
https://arcade.makecode.com/app/497e4b372fbf2a00029b6003585e4f05603e2a05-d864d31504---skillmap

**Comment by riknoll** [comment]
no change

**Comment by riknoll** [comment]
The "isFirefox" check returns false on mac (I just debugged it)

**Comment by shakao** [comment]
(i think maybe that build doesn't have the latest changes? it isn't using the isFirefox check any more)

**Comment by livcheerful** [comment]
whatttt that upload shouldn't use isfirefox though. maybe i uploaded wrong hmmmmm

**Pull request merged**